pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -692,12 +692,15 @@ func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podContainerC
 		enforceCPULimits = false
 		klog.V(2).InfoS("Disabled CFS quota", "pod", klog.KObj(pod))
 	}
-	podResources := cm.ResourceConfigForPod(pod, enforceCPULimits, uint64((m.cpuCFSQuotaPeriod.Duration)/time.Microsecond), false)
+	cpuPeriod := uint64((m.cpuCFSQuotaPeriod.Duration) / time.Microsecond)
+	podResources := cm.ResourceConfigForPod(pod, enforceCPULimits, cpuPeriod, false)
 	if podResources == nil {
 		klog.ErrorS(nil, "Unable to get resource configuration", "pod", pod.Name)
 		result.Fail(fmt.Errorf("unable to get resource configuration processing resize for pod %s", pod.Name))
 		return
 	}
+
+	const maxValue = int64(-1)
```

Comment:
I see in your comment https://github.com/kubernetes/kubernetes/pull/127143#discussion_r1873336176 that this doesn't appear to be documented anywhere, maybe it'll be helpful to leave a comment here with this context so that in the future we can remember where this constant came from

Reply:
I added a comment.